classes/Leth/IPAddress/IPv6/Address.php

```diff
@@ -237,6 +232,12 @@ public function format($mode)
 				}
 				break;
 
+			case IPv6\Address::FORMAT_MAPPED_IPV4:
```

Comment:
It's the kind of thing you might want in display code, or in the `toString` method